UMLClass.py

```diff
@@ -77,7 +77,8 @@ def renameClass(oldName: str, newName: str):
     else:
         print("Rename failed")
 
-
+def classIndexx():
```

Comment:
This kinda seems redundant, is there a benefit to adding it?

Reply:
no benefit just wanted a function call to get the list I know we can use UMLClass.classIndex,when debugging i added that for myself it can be removed